dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/context/DubboSpringInitializer.java

```diff
@@ -68,14 +68,15 @@ public static boolean remove(BeanDefinitionRegistry registry) {
     }
 
     public static boolean remove(ApplicationContext springContext) {
+        AutowireCapableBeanFactory autowireCapableBeanFactory = springContext.getAutowireCapableBeanFactory();
         for (Map.Entry<BeanDefinitionRegistry, DubboSpringInitContext> entry : contextMap.entrySet()) {
             DubboSpringInitContext initContext = entry.getValue();
             if (initContext.getApplicationContext() == springContext ||
-                initContext.getBeanFactory() == springContext.getAutowireCapableBeanFactory() ||
-                initContext.getRegistry() == springContext.getAutowireCapableBeanFactory()
+                initContext.getBeanFactory() == autowireCapableBeanFactory ||
+                initContext.getRegistry() == autowireCapableBeanFactory
             ) {
                 DubboSpringInitContext context = contextMap.remove(entry.getKey());
-                logger.info("Unbind " + safeGetModelDesc(context.getModuleModel()) + " from spring container: " +
+                logger.info("Unbind " + context.getModuleModel().getDesc() + " from spring container: " +
```

Comment:
Is it possible throws NPE here?

Reply:
> Is it possible throws NPE here?

The initContext method ensures that the context.getModuleMode() does not return null.

Reply:
> Remove before init may erorr?

right